product_app/models.py

```diff
@@ -0,0 +1,148 @@
+from django.db import models
+
+class Menu(models.Model):
+    name = models.CharField(max_length = 100)
+    
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        db_table = 'menus'
+    
+class MenuCategory(models.Model):
+    menus      = models.ForeignKey(Menu, on_delete = models.CASCADE)
+    categories = models.ForeignKey('Category', on_delete = models.CASCADE)
+
+class Category(models.Model):
+    name  = models.CharField(max_length = 100)
+    menus = models.ManyToManyField(Menu, through = 'MenuCategory')
+    
+    def __str__(self):
+        return f'name: {self.name}, menus: {menus}'
+    class Meta:
+        db_table = 'categories'
+
+class CategorySubcategory(models.Model):
+    categories     = models.ForeignKey(Category, on_delete = models.CASCADE)
+    sub_categoreis = models.ForeignKey('SubCategory', on_delete = models.CASCADE)
+
+class SubCategory(models.Model):
+    name         = models.CharField(max_length = 100)
+    descriptions = models.CharField(max_length = 1000)
+    categories   = models.ManyToManyField(Category, through = 'CategorySubcategory')
+    
+    def __str__(self):
+        return f'name: {self.name}, categories: {self.categories}'
+    class Meta:
+        db_table = 'sub_categories'
+
+
+class Size(models.Model):
+    name = models.CharField(max_length=20, default = 0, unique = True)
+    
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        db_table = 'sizes'
+
+
+class Material(models.Model):
+    name = models.CharField(max_length = 100, unique = True)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        db_table = 'materials'
+
+
+class Country(models.Model):
+    name = models.CharField(max_length = 100, unique = True)
+
+    class Meta:
+        db_table = 'countries'
+
+
+class Product(models.Model):
+    name           = models.CharField(max_length = 200, unique = True)
+    price          = models.IntegerField()
```

Comment:
가격은 DecimalField로 해야합니다. 국제통화룰로 뒤에 .00 소수점 두자리 유지해주세요.
```suggestion
    price          = models.DecimalField(max_digits=12, decimal_places=2)
```